src/igvfd/tests/test_views.py

```diff
@@ -163,23 +164,24 @@ def test_item_actions_filtered_by_permission(testapp, authenticated_testapp, lab
     assert not any(action for action in res.json.get('actions', []) if action['name'] == 'edit')
 
 
-def test_collection_put(testapp, execute_counter):
+def test_collection_put(testapp, execute_counter, pi, unverified_member):
     initial = {
         'name': 'lab-a',
-        'title': 'The Lab A',
-        'address1': 'xyz'
+        'institute_label': 'Institute A',
+        'pi': pi['@id'],
     }
     item_url = testapp.post_json('/lab', initial).location
-
-    with execute_counter.expect(1):
+    print(item_url)
```

Comment:
There seems to be a print statement here that maybe was a debug?

Reply:
Yes it was. Removed.